kubernetes/gaffer/values.yaml

```diff
@@ -84,31 +76,31 @@ test:
     curl:
       repository: curlimages/curl
       tag: 7.67.0
-
-
 # Extras
-
 accumulo:
   enabled: false
   image:
     repository: gchq/gaffer
-    tag: 1.21.1
+    tag: 1.21.1 # managed version
   config:
     postInstallCommands:
-    - users
-    - userpermissions -u gaffer
-    - getauths -u gaffer
+      - users
+      - userpermissions -u gaffer
+      - getauths -u gaffer
     userManagement:
       users:
         gaffer:
           password: "gaffer"
           auths: []
           permissions:
             system:
-            - CREATE_TABLE
+              - CREATE_TABLE
             table:
               simpleGraph:
-              - READ
-              - WRITE
-              - BULK_IMPORT
-              - ALTER_TABLE
+                - READ
+                - WRITE
+                - BULK_IMPORT
+                - ALTER_TABLE
+testImages:
```

Comment:
This was also added by yq but I am not sure why?

Reply:
Yeah, I think it's left over from before the integration tests got changed. I've removed it.